activities.json

```diff
@@ -933,6 +933,19 @@
     "title": "Mixed Content",
     "url": "https://w3c.github.io/webappsec-mixed-content/"
   },
+  {
+    "ciuName": null,
+    "description": "The Network Error Logging spec enables website to declare a reporting policy that can be used to report encountered network errors that prevented it from successfully loading its requested resources.",
+    "id": "network-error-logging",
+    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1145235",
+    "mozPosition": "negative",
+    "mozPositionDetail": "The API enables the collection of user-specific information that sites might not otherwise be able to observe, which includes information that might be private under some circumstance. Furthermore, the specification does not seem to track changes to the Reporting API it builds upon and seems effectively unmaintained.",
```

Comment:
```suggestion
    "mozPositionDetail": "The API enables the collection of user-specific information that sites might not otherwise be able to observe, which includes information that might be private under some circumstances. Furthermore, the specification does not seem to track changes to the Reporting API it builds upon and seems effectively unmaintained.",
```